src/utils/api.js

```diff
@@ -10,4 +10,12 @@ export const getReviews = () => {
     .then((res) => {
         return res.data
     })
+}
+
```

Comment:
On line 4 of this file could we change http -> https in the baseURL.  This sometimes causes issues with the request being rejected for being insecure.  Currently I am just seeing 'Loading...' because the request is being rejected in my browser